### ISSUE TITLE
VideoCaptureFormat width/height is not exposed inside Objective C/Swift headers

### ISSUE THREAD

**Issue by nealmanaktola** [aws/amazon-chime-sdk-ios#240]
**Is your feature request related to a problem? Please describe.**
Hello, I am trying to specify the VideoCaptureFormat with a specified resolution
https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/custom_video.md

```
   // Switch the camera
    cameraCaptureSource.switchCamera()

    // Get the current device and format
    let currentDevice = cameraCaptureSource.device
    let currentFormat = cameraCaptureSource.format

    // Pick a new device explicitly 
    let newDevice = MediaDevice.listVideoDevices().first { mediaDevice in
        mediaDevice.type == MediaDeviceType.videoFrontCamera
    }
    cameraCaptureSource.device = newDevice

    // Pick a new format explicitly (reverse these so the highest resolutions are first)
    let newFormat = MediaDevice.listSupportedVideoCaptureFormats(mediaDevice: videoDevice).reversed().filter { $0.height < 800 }.first
    if let format = newFormat {
        cameraCaptureSource.format = format
    }
```
Unfortunately on Objective C,  I can't inspect the width or height property on `VideoCaptureFormat`
If i look at the interface for VIdeoCaptureFormat on file `AmazonChimeSDK-Swift.h`

```
/// <code>VideoCaptureFormat</code>describes a given capture format that may be possible to apply to a <code>VideoCaptureSource</code>.
/// Note that <code>VideoCaptureSource</code> implementations may ignore or adjust unsupported values.
SWIFT_CLASS("_TtC14AmazonChimeSDK18VideoCaptureFormat")
@interface VideoCaptureFormat : NSObject
- (BOOL)isEqual:(id _Nullable)object SWIFT_WARN_UNUSED_RESULT;
- (nonnull instancetype)init SWIFT_UNAVAILABLE;
+ (nonnull instancetype)new SWIFT_UNAVAILABLE_MSG("-init is unavailable");
@end
```

looks the properties are missing on the objective c/swift headers? 

**Describe the solution you'd like**
Please expose width height fields

**Additional context**
Using a fork of the react-native app

**Comment by zhinang-amazon** [comment]
Hi @nealmanaktola,

Thanks for reporting this issue. I'm able to reproduce this issue in ObjC. I'm working on the fix and will update this issue once we release the fix.

**Comment by dylonChime** [comment]
The issue was fixed with most recent release (0.16.1). Feel free to reopen if the issue re-emerges.
